packages/components/src/components/Cell/index.ts

```diff
@@ -15,5 +16,6 @@ export {
     Cell,
     CellProps,
     CellGroupProps,
-    NavigationCellProps
+    NavigationCellProps,
+    CellSwipeItemProps as CellSwipeGroup
```

Comment:
I don't know how I feel about renaming `CellSwipeItemProps`.

Reply:
If you import from index, it is called CellSwipeGroup, but if you import from types, it is called CellSwipeItemProps. But the types are identical. This can be confusing

Reply:
fixed in 06e1591